MaxText/configs/models/gpt3-dummy.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Google LLC
+#
```

Comment:
I don't know what dummy means (but you have a nice comment). gpt3-style-1B or something would be better.

Reply:
Will change it to `gpt3-52k` instead. 
It is an extremely small 52k parameter model for [numerical test usage](https://github.com/google/maxtext/pull/293/commits/62800feb62a9c01e640d40fa3e6caa4dcab81ee9).